tools/run_defacing_script.pl

```diff
@@ -318,18 +318,18 @@ =head3 grep_FileIDs_to_deface($session_id_arr, $modalities_to_deface_arr)
 =cut
 
 sub grep_FileIDs_to_deface {
-    my ($session_id_arr, $modalities_to_deface_arr) = @_;
+    my ($session_id_arr, @modalities_to_deface_arr) = @_;
```

Comment:
I would rename these variables:
- `$session_id_arr` should be `$session_ids_ref` (it's a reference)
- `@modalities_to_deface_arr` should be `@modalities_to_deface`